docs/content/webhooks-app.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Webhooks App
+metaDescription: Effortlessly set up and send webhooks with our comprehensive guide.
+---
+
+# Setting Up the App Instance
+
+Begin by installing the webhooks app from the Studio App store. Once you've named your instance, you'll get a unique webhook URL and an API key, both essential for sending webhook requests.
+
+Here's what your unique webhook URL will look like:
+
+```
+https://apps-api-feeds.{region}.screencloudapps.com/v1/hooks/{uniqueId}
```

Comment:
Point of clarity here - we haven't got versioning in the feeds repo, so there will be no v1 here - the URL is just `https://apps-api-feeds.eu.staging.screencloudapps.com/hooks/{uniqueId}`

Do we need to think about implementing versioning here, or "faking" it to add a fixed `v1` into the hooks endpoint?

Reply:
I think we should add the `v1` now, it can be fake, but as soon as you release this API without it, then customers will be stuck on that invalid URL. So we deffo want to add `/v1` now, but it don't need to do anything.

Reply:
👍 I've [raised a ticket](https://github.com/screencloud/apps/issues/10154) for this and self-assigned it - will hopefully get to it soon.

Reply:
Good spot @peteraikensc sorry I missed this on your PR. Yeah we are faking the version here even though the code underneath doesn't actually use the version.